packages/api-reference/src/features/test-request-button/TestRequestButton.vue

```diff
@@ -34,9 +35,9 @@ const handleClick = () => {
     :method="operation.method"
     type="button"
     @click.stop="handleClick">
-    <ScalarIcon
-      icon="Play"
-      size="sm" />
+    <ScalarIconPlay
```

Comment:
This will also affect the modern layout (and everywhere else we use the test request button).

<img width="2980" height="1968" alt="Google Chrome-2025-07-11-21-44-52@2x" src="https://github.com/user-attachments/assets/87a4b4ac-c7b7-45b9-8a2f-42c2510add20" />